cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetector.java

```diff
@@ -58,19 +60,23 @@
   private final DiskFailureDetector _diskFailureDetector;
   private final ScheduledExecutorService _detectorScheduler;
   private final long _anomalyDetectionIntervalMs;
-  private final LinkedBlockingDeque<Anomaly> _anomalies;
+  private final PriorityBlockingQueue<Anomaly> _anomalies;
   private volatile boolean _shutdown;
   private final AnomalyDetectorState _anomalyDetectorState;
   private final List<String> _selfHealingGoals;
   private final ExecutorService _anomalyLoggerExecutor;
-  private volatile Anomaly _anomalyInProgress;
+  private volatile KafkaAnomaly _anomalyInProgress;
```

Comment:
Why are we using `KafkaAnomaly` rather than `Anomaly`? -- i.e. `KafkaAnomaly` is just one implementation of `Anomaly`.
Are there remaining changes needed in the interface to continue using the generic interface type?

Reply:
It is missed in the effort to revert `KafkaAnomaly` to `Anomaly`.